### ISSUE TITLE
check docker image synch with fiware docker hub

### ISSUE THREAD

**Issue by chicco785** [Atos-Research-and-Innovation/IoTagent-LoRaWAN#158]
@jason-fox can you let us know if any update is needed to ensure that fiware images in the docker repo are up to date?

**Comment by jason-fox** [comment]
Since Docker removed the free tier tracking of Docker builds, this is currently a manual process for the FIWARE Images.

The latest FIWARE image is 1.2.3 - see https://hub.docker.com/r/fiware/iotagent-lorawan so it appears that one release has been missed by  `fiware/iotagent-lorawan`. I'll have to kick of the script to build it.

**Comment by chicco785** [comment]
hi @jason-fox can't we automated this using a github action? so that also latest and edge images are pushed?

**Comment by jason-fox** [comment]
Yes, probably. I've done something like that [here](https://github.com/FIWARE/iotagent-isoxml/blob/master/.github/workflows/baseimage.yml) in a repo under the FIWARE GitHub, but I can't share direct push access to the FIWARE Docker Hub to a repo which isn't under our direct control.

The next step would be to either take the `ghcr.io` parts from that action and add them here, or to generate the docker hub images onto an ATOS account over on Docker Hub.

**Comment by chicco785** [comment]
We fixed the generation of images on top of the docker hub account of atos,
see the workflow here:
https://github.com/Atos-Research-and-Innovation/IoTagent-LoRaWAN/blob/master/.github/workflows/docker.yml

**Comment by arturohu** [comment]
Where could I find the Docker images of each tag?
